src/components/inline/Button/Button.js

```diff
@@ -3,18 +3,59 @@ import React from "react";
 
 import s from "./Button.module.css";
 
-const Button = ({ children }) => (
-  <button className={s.button} type="button">
-    {children}
-  </button>
-);
+const Button = ({ button }) => {
+  let buttonView;
 
-Button.propTypes = {
-  children: PropTypes.node,
+  if (button.externalLink) {
+    buttonView = (
+      <a rel="noreferrer" target="_blank" href={button.externalLink}>
+        <button type="button" className={s.button}>
+          {button.text}
+        </button>
```

Comment:
For external links, I would suggest not using the `<button>` element at all, and instead just style the `<a>` tag directly [to look like a button](https://css-tricks.com/css-basics-styling-links-like-boss/#links-as-buttons).

Reply:
Me and Richard Agreed on the a11y conventions and kept button.